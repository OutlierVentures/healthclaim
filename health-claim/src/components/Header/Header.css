.navbar {
  background: linear-gradient(#0076ba, #56c1ff);
}

.header {
  color: white;
  font-size: 1.7em;
  font-weight: bold;
}
