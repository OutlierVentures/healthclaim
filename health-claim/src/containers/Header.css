.navbar {
    background:linear-gradient(#0076BA, #56C1FF);
}

.header {
    color: white;
    font-size: 1.7em;
    font-weight: bold;
}