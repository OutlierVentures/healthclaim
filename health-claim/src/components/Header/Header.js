import React from "react";
import "./Header.css";
import { Navbar, Nav, NavItem } from "reactstrap";

const Header = () => (
  <div>
    <Navbar className={"navbar"} light expand={"md"}>
      <Nav>
        <NavItem>
          {" "}
          <img src="./icon.png" alt="logo" />
        </NavItem>
        <NavItem className="header">HEALTH CLAIMS</NavItem>
      </Nav>
    </Navbar>
  </div>
);

export default Header;