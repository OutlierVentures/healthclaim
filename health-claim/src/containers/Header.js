import React, { Component } from 'react';
import './Header.css';
import {
  Navbar, Nav, NavItem
} from 'reactstrap';

export default class RecordCard extends Component {
  
  render() {
    return (
      <div>
        <Navbar className={"navbar"} light expand={"md"}>
          <Nav>
            <NavItem> <img src="./icon.png" alt="logo"/></NavItem>
            <NavItem className="header">HEALTH CLAIMS</NavItem>
          </Nav>
        </Navbar>
      </div>
    )
  }
}