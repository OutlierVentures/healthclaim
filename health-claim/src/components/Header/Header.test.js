import React from "react";
import renderer from "react-test-renderer";
import Header from "./Header";

describe("Header", () => {
  it("should render correctly", () => {
    const header = renderer.create(<Header />).toJSON();
    expect(header).toMatchSnapshot();
  });
});
