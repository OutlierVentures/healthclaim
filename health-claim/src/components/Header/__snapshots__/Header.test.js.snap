// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<div>
  <nav
    className="navbar navbar navbar-expand-md navbar-light"
  >
    <ul
      className="nav"
    >
      <li
        className="nav-item"
      >
         
        <img
          alt="logo"
          src="./icon.png"
        />
      </li>
      <li
        className="header nav-item"
      >
        HEALTH CLAIMS
      </li>
    </ul>
  </nav>
</div>
`;
